app/routes/events/EventEditRoute.ts

```diff
@@ -29,10 +30,13 @@ import {
 } from './utils';
 
 const mapStateToProps = (state, props) => {
-  const eventId = props.match.params.eventId;
-  const event = selectEventById(state, {
-    eventId,
-  });
+  const eventIDorSlug = props.match.params.eventId;
```

Comment:
```suggestion
  const eventIDorSlug = props.match.params.eventIDorSlug;
```
Hasn't this changed because of path change in the `RouteWrapper`?

Reply:
Yes